test/services/ProfessionalBodiesServiceSpec.scala

```diff
@@ -29,72 +29,30 @@ class ProfessionalBodiesServiceSpec extends SpecBase with MockitoSugar with Scal
 
   private val professionalBodiesService = new ProfessionalBodiesService(Environment.simple(), frontendAppConfig)
 
+  // TODO: Move this into an integration test suite
```

Comment:
Do we think that this is worth doing? It does interact with the file system

Reply:
We've decided to not move this into a separate test suite since it is a core part of the system